server/src/schemas/storeFrontSchema.ts

```diff
@@ -0,0 +1,74 @@
+import { gql } from "apollo-server-express";
+
+export const StoreFrontTypeDefs = gql`
+
+	scalar Date
+
+	type InventoryItem {
+		id: ID!;
+		image: String;
+		name: String;
+		labels: [String];
+		unit: String;
+		pluralUnit: String;
+		count: Int;
+		pricePerUnit: Float;
+	}
+
+	input InventoryItemInput {
+		image: String;
+		name: String;
+		labels: [String];
+		unit: String;
+		pluralUnit: String;
+		count: Int;
+		pricePerUnit: Float;
+	}
+
+	type PurchaseOrderItem {
+		id: ID!;
+		item: InventoryItem;
+		count: Int;
+	}
+
+	type PurchaseOrder {
+		id: ID!;
+		creator: Person;
+		createDate: Date;
+		expectedDeliveryDate: Date;
+		items: [PurchaseOrderItem];
+		attachments: [String];
+	}
+
+	input PurchaseOrderInput {
+		creator: Person;
+		createDate: Date;
```

Comment:
`createDate` will be generated by the system, not inputted by the user, no?

Reply:
Fair point